Resources/Prototypes/_Moffstation/Entities/Mobs/Customization/Markings/reptilian_markings.yml

```
@@ -0,0 +1,15 @@
+- type: marking
+  id: ReptilianDragonEye
+  bodyPart: Eyes
+  markingCatagory: Overlay
+  speciesResriction: [Reptilian]
+  coloring:
+    default:
+      type: 
+        !type:EyeColoring
+          negative: true 
```

```suggestion
      type: !type:EyeColoring
        negative: true 
```
The syntax for this is a little bit odd, but the `!type:Whatever` is a "tag" which describes the entire yaml object and is not _part_ of the object. For this reason, it's canonically organized like this.